base/linalg/lapack.jl

```diff
@@ -1355,19 +1357,19 @@ for (gelsd, gelsy, elty, relty) in
                 ccall((@blasfunc($gelsy), liblapack), Void,
                     (Ptr{BlasInt}, Ptr{BlasInt}, Ptr{BlasInt}, Ptr{$elty},
                      Ptr{BlasInt}, Ptr{$elty}, Ptr{BlasInt}, Ptr{BlasInt},
-                     Ptr{$elty}, Ptr{BlasInt}, Ptr{$elty}, Ptr{BlasInt},
```

Comment:
`Ref` helped me catch this bug/error.  The `ccall` signature here says it's a Ptr to an `$elty` but in fact `rcond` is a `$relty`